solr/solrj/src/java/org/apache/solr/common/annotation/SolrSingleThreaded.java

```diff
@@ -29,6 +29,6 @@
 @Documented
 @Retention(SOURCE)
 @Target(TYPE)
-public @interface SolrSingleThreaded {
+public @interface SolrThreadUnsafe {
```

Comment:
Please change the name of the file too.

Reply:
oops

Reply:
stepped away to get dinner. back now

Reply:
@anshumg Should be good now. Mid-work I was starting to get headache from now eating.